src/styles/utils/mixins.scss

```
@@ -0,0 +1,15 @@
+@mixin contentFlexCenterCenter() {
```

```suggestion
@mixin contentFlexCenter {
```
you may remove brackets if mixin doesn't accept any argument

---

Thank you for recommendations!